### PR TITLE
Refactor the Ragel machines' code

### PR DESCRIPTION
**Pull request by Anishka0107**
Small refactoring in the ragel rules

### CODE REVIEW

File: src/vcf/vcf_v41.ragel [hunk 1]
```diff
@@ -74,36 +74,36 @@
     ## - The line is a single key-value pair (meta_value): the value cannot start with an angle bracket, quote or linebreak, but it is free text thereafter
     ## - One of multiple key-value pairs, non-quoted (meta_field_value): the value cannot contain a comma -end of field-, or a closing angle bracket -end of entry-, or quote
     ## - One of multiple key-value pairs, quoted (meta_field_desc): the value can contain any character except for non-escaped quotes
-    meta_value      = ((print - ("<" | "\"" | NL)) (print)*)	>token_begin $token_middle %token_end ;
-    meta_field_value= (print - ("," | ">" | "\""))+		>token_begin $token_middle %token_end ;
-    meta_field_desc = (("\\" print) | (print - "\""))*		>token_begin $token_middle %token_end ;
+    meta_value      = ((print - ("<" | '"' | NL)) (print)*)	>token_begin $token_middle %token_end ;
```
Comment (cyenyxe) [inline src/vcf/vcf_v41.ragel]: All the strings should be surrounded by single quotes, not only those that require escaping. Having some with single and some with double quotes will make the machine definition look inconsistent.

Comment (Anishka0107) [inline src/vcf/vcf_v41.ragel]: Oh yes! I'll do it

Comment (Anishka0107) [inline src/vcf/vcf_v41.ragel]: Done in next commit

File: src/vcf/vcf_v41.ragel [hunk 2]
```diff
@@ -58,140 +58,140 @@
...
     ## A value can be expressed in multiple ways:
     ## - The line is a single key-value pair (meta_value): the value cannot start with an angle bracket, quote or linebreak, but it is free text thereafter
     ## - One of multiple key-value pairs, non-quoted (meta_field_value): the value cannot contain a comma -end of field-, or a closing angle bracket -end of entry-, or quote
     ## - One of multiple key-value pairs, quoted (meta_field_desc): the value can contain any character except for non-escaped quotes
-    meta_value      = ((print - ("<" | "\"" | NL)) (print)*)	>token_begin $token_middle %token_end ;
-    meta_field_value= (print - ("," | ">" | "\""))+		>token_begin $token_middle %token_end ;
-    meta_field_desc = (("\\" print) | (print - "\""))*		>token_begin $token_middle %token_end ;
+    meta_value      = ((print - ('<' | '"' | NL)) (print)*)	>token_begin $token_middle %token_end ;
+    meta_field_value= (print - (',' | '>' | '"'))+		>token_begin $token_middle %token_end ;
+    meta_field_desc = (('\\' print) | (print - '"'))*		>token_begin $token_middle %token_end ;
```
Comment (jmmut) [inline src/vcf/vcf_v41.ragel]: do we have some test file that uses this `'\\'` rule?

Comment (Anishka0107) [inline src/vcf/vcf_v41.ragel]: Yes, I guess we do.
Passed: https://github.com/EBIvariation/vcf-validator/blob/develop/test/input_files/v4.3/passed/passed_meta_format.vcf#L3
Failed: https://github.com/EBIvariation/vcf-validator/blob/develop/test/input_files/v4.2/failed/failed_meta_001.vcf#L3

**Pull request merged**